src/frontend/src/app/search/search.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import {ProjectDto} from "../model/projectDto";
+import {SearchService} from "./search.service";
+import {ActivatedRoute, Params, Router} from '@angular/router';
+import {IssueDto} from "../model/IssueDto";
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  providers: [SearchService],
+  styleUrls: ['./search.component.css']
+})
+export class SearchComponent implements OnInit {
+
+  projects: ProjectDto[] = [];
+  issues: IssueDto[] = [];
+
+  results: string;
+
```

Comment:
jeden odstęp między konstruktorem jest niepotrzebny i również zapomniałeś o modyfikatorach dostępu (public jest default ale lepiej pisać to :) )